style_editor.html

```diff
@@ -1432,12 +1432,32 @@
 
 // Create n textures of about 1MB each.
 function initGL() {
+  loadColorData();
+  // Clear existing tab links and tab bodies before populating
+  var tabLinksElement = FIND("TABLINKS");
+  var tabBodiesElement = FIND("TABBODIES");
```

Comment:
Instead of clearing out all the tabs all the time, we should just re-populate the color tab when needed.
We can make a copy of AddTabContent(), but change the += to = and call it SetTabContent(), then use SetTabContent("rgb", ......) to update the colors tab when needed.

Reply:
see below https://github.com/profezzorn/ProffieOS-StyleEditor/pull/139/files#r1436356076